boost-maven/boost-boms/booster-ee8-bom/pom.xml

```diff
@@ -60,12 +60,12 @@
 				<scope>provided</scope>
 			</dependency>
 			<dependency>
-				<groupId>io.openliberty.boosters</groupId>
+				<groupId>boost.boosters</groupId>
 				<artifactId>cdi</artifactId>
 				<version>0.2-SNAPSHOT</version>
 			</dependency>
 			<dependency>
-				<groupId>io.openliberty.boosters</groupId>
+				<groupId>boost.boosters</groupId>
 				<artifactId>jsonp</artifactId>
 				<version>0.2-SNAPSHOT</version>
```

Comment:
'jsonp' dependency isn't really doing anything here, at least not in Liberty (probably not in TomEE either).   Maybe slightly confusing to leave it in not a huge deal though.

We should start looking at using 'jsonp' together with a JSON provider dependency to use a non-default provider, as a totally separate issue though.